config_parser_test.cc

```diff
@@ -9,3 +9,34 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+//Fixture implemented
+class NginxStringConfigParserTest : public ::testing::Test{
+protected:
+bool ParseString (const std::string config_string){
+  std:: stringstream config_stream(config_string);
+  return parser_.Parse(&config_stream, &out_config_);
+}
+NginxConfigParser parser_;
+NginxConfig out_config_;
+};
```

Comment:
style issue: you should indent the methods and member variables, and remove the space after `std::` on line 17.